pkg/reaper/labels.go

```diff
@@ -14,18 +16,14 @@ var commonLabels = map[string]string{
 }
 
 func createResourceLabels(kc *k8ssandraapi.K8ssandraCluster) map[string]string {
-	labels := map[string]string{
-		k8ssandraapi.K8ssandraClusterNameLabel:      kc.Name,
-		k8ssandraapi.K8ssandraClusterNamespaceLabel: kc.Namespace,
-		k8ssandraapi.CreatedByLabel:                 k8ssandraapi.CreatedByLabelValueK8ssandraClusterController,
-	}
-	return utils.MergeMap(commonLabels, labels)
+	return utils.MergeMap(
+		commonLabels,
+		k8ssandralabels.CleanedUpByLabels(client.ObjectKey{Namespace: kc.Namespace, Name: kc.Name}))
 }
 
 func getConstantLabels(r *reaperapi.Reaper) map[string]string {
 	labels := map[string]string{
-		reaperapi.ReaperLabel:       r.Name,
-		k8ssandraapi.CreatedByLabel: k8ssandraapi.CreatedByLabelValueReaperController,
+		reaperapi.ReaperLabel: r.Name,
```

Comment:
This is used in the service's selector, I've checked that removing a label from it is not a breaking change.